airbyte-cdk/python/CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ Fix CDK version mismatch introduced in 0.78.8
 Update error messaging/type for missing streams. Note: version mismatch, please use 0.78.9 instead
 
 ## 0.78.6
-low-code: add backward compatibility for old close slice behavior 
+low-code: add backward compatibility for old close slice behavior
```

Comment:
were changes to this file intentional?

Reply:
I think these were unintentional -- I didn't realize there was a Publish CDK action so I had previously manually updated the changelog/pyproject, then reverted the changes.

Reply:
ok please only revert changes to this file. It'll be updated as part of the publish action

Reply:
Updated.